Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/util/CoreHelpUtils.java

```diff
@@ -94,4 +96,16 @@ public static XMLGregorianCalendar getXMLGregorianCalendarFrom(Date date) {
         return retList;
     }
 
+    public static Date getDate(XMLGregorianCalendar xmlDate) {
+        return null != xmlDate ? xmlDate.toGregorianCalendar().getTime() : null;
+    }
+
+    public static boolean isId(Long id) {
+        return null != id && id.longValue() > 0;
+    }
+
+    public static List<String> sort(List<String> list) {
```

Comment:
Just curious, why would you need to add this method

Reply:
Well you can just use Collection.sort() to sort your list .. you don't need this method